README.md

```diff
@@ -1,29 +1,27 @@
 # History #
-Every time i get away from my computer , due to privacy concern
-i had to log out whatsapp from computer and again login when i reuse
-this was really annoying ,
+Every time I move away from my computer, due to privacy concerns
+I had to log out of whatsapp from my computer, and then had to login again when i want to reuse it.
+This was really annoying,
 <br>
 so i decided to make this little app
 <br>
 
 # Introduction #
-<b>Locked Whatsapp</b> is simply wrapper around https://web.whatsapp.com
+<b>Locked Whatsapp</b>, is simply a wrapper around https://web.whatsapp.com,
 but supports password protection.
-To access your web whatsapp account you will have to setup password.
+To access your web whatsapp account you will have to set a password.
 
  ##### Usage #####
-For the first time you open this app this will ask you 
-password to lock the app and after that it will be unlocked only
-using that password.
+When you open the app for the first time, it will ask you to create a password to lock the app. 
+Thereafter, the app will only be opened using the password you've created.
 
 <br>
-Incase you forget The password You can click on Reset 
-it will automatically log you out from whatsapp
-and again ask you a new password
+Should you forget your password, you can click on Reset, which will autonatically log you out from
```

Comment:
automatically*

Reply:
@daudmalik06 .. Thanks, missed that one, updating the file now.